DOC: instruction for test env setup (#167)
enthought/scimath: README.rst

* `Traits <https://pypi.python.org/pypi/traits>`_
* `NumPy <https://pypi.python.org/pypi/numpy>`_
* `SciPy <https://pypi.python.org/pypi/scipy>`_

Development Environment Setup
-------------

> 1. Setup the EDM (with click, setuptools and coverage): edm install -e bootstrap click, setuptools, coverage

Author: Hi Mark, which command wasn't working? I just tested the procedure (running the instructions consecutively), the output seems fine (On MacOS Monterey 12.6.3):
<img width="733" alt="Screen Shot 2023-03-14 at 2 42 37 PM" src="https://user-images.githubusercontent.com/102019577/224907139-5aa3ac00-3839-486f-99b3-495cbe6b4d4b.png">

> 1. Setup the EDM (with click, setuptools and coverage): edm install -e bootstrap click, setuptools, coverage

Reviewer: Apologies. It's not normal to put commas in the EDM install list, and I mistakenly assumed that this wouldn't work at the EDM prompt, but apparently it does. That is, the normal form is:
```
edm install -e name_of_your_scimath_env click setuptools coverage
```
rather than
```
edm install -e name_of_your_scimath_env click, setuptools, coverage
```

* `Traits <https://pypi.python.org/pypi/traits>`_
* `NumPy <https://pypi.python.org/pypi/numpy>`_
* `SciPy <https://pypi.python.org/pypi/scipy>`_

Development Environment Setup
-----------------------------

> To setup the EDM environment for this project::

Reviewer: A couple of grammar nits:

```suggestion
To set up an EDM environment for this project::
```

>     $ edm install -e name_of_your_scimath_env click, setuptools, coverage

Reviewer: Let's use the form without the commas.

```suggestion
    $ edm install -e name_of_your_scimath_env click setuptools coverage
```

Reviewer: Can you please double-check this list? We definitely need click, and we might also need `setuptools`, but I'd be surprised if we need `coverage`.

Author: I just checked, the coverage is indeed not necessary, removed. And it seems not need setuptools as well as tested separately... I just push a new version with format update and these two removed
